src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -0,0 +1,82 @@
+@import "scss/constants.scss";
+
+.ScheduleEvent {
+  display: flex;
+  position: absolute;
+  border-radius: 18px;
+  padding: 0 4px 0 12px;
+  cursor: pointer;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #3f3d42;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.65);
+  margin-top: 0.25rem;
```

Comment:
these values should be imported from the `constants`

Reply:
refactored styles here: https://github.com/sparkletown/sparkle/pull/1302/commits/06620c6fb5ce10998508b5403d8b8cf907ed32d6